libnmstate/nm/connection.py

```diff
@@ -26,11 +26,17 @@
 
 
 class ConnectionProfile:
+    _ACTIVATION_CHECK_INTERNAL = 1  # second
+    _MAX_ACTIVATION_CHECK_COUNT = 34
+    # The maximum activation status check count in seconds.
+    # Context: The STP need 30 seconds, and nmstate mainloop timeout is 35.
```

Comment:
If it depends on another static, maybe it needs to be expressed using an arithmetic operation.
I am also not sure I understand how the timeout is affected by these other limitations.